packages/esm-patient-registration-app/src/patient-registration/input/custom-input/identifier/identifier-input.component.tsx

```diff
@@ -18,15 +18,17 @@ interface IdentifierInputProps {
 }
 
 const IdentifierInput: React.FC<IdentifierInputProps> = ({ patientIdentifier, fieldName }) => {
+  const { autoGeneration, initialValue, identifierValue, identifierName, required, selectedSource } = patientIdentifier;
   const { t } = useTranslation();
-  const { defaultPatientIdentifierTypes } = useConfig();
+  const { defaultPatientIdentifierTypes, fieldConfigurations } = useConfig();
   const { identifierTypes } = useContext(ResourcesContext);
   const { isOffline, values, setFieldValue } = useContext(PatientRegistrationContext);
   const identifierType = useMemo(
     () => identifierTypes.find((identifierType) => identifierType.uuid === patientIdentifier.identifierTypeUuid),
     [patientIdentifier, identifierTypes],
   );
-  const { autoGeneration, initialValue, identifierValue, identifierName, required } = patientIdentifier;
+  const manualEntryEnabled =
+    fieldConfigurations.id.allowAutoGenerationManualEntry && selectedSource?.autoGenerationOption?.manualEntryEnabled;
```

Comment:
Is there reason we need to add the new O3 config property here (ie "allowAutoGenerationManulEntry") instead of entirely relying on the "autogenerationOption.manualEntryEnabled" config that comes from the IDGen module?

Reply:
@mogoodrich The config property is because I was not sure if this feat should be enabled by default. It can be removed.

Reply:
I'm not sure we need extra configuration here. Makes sense to remove it.